LondonBritishLibrary/orient/BLorient614.xml

```diff
@@ -0,0 +1,249 @@
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2">
+<locus from="136r"/>
+<title xml:lang="en" type="complete">Hymns to the archangels</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.1">
+<locus from="136r"/>
+<title xml:lang="en" type="complete" ref="LIT2835RepCh119">To Michael</title>
+<incipit xml:lang="gez">ሰላም፡ ለዝክረ፡ ስምከ፡ ምስለ፡ ስም፡ ልዑል፡ ዘተሳተፈ። ወልደ፡ ያሬድ፡ ሄኖክ፡ በከመ፡ ጸሐፈ።</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.2">
+<locus from="144r"/>
+<title xml:lang="en" type="complete" ref="LIT2849RepCh131">To Raphael</title>
+<incipit xml:lang="gez">ሰላም፡ ለዝክረ፡ ስምከ፡ ተጸውዖ፡ ዘቀደመ። በአፈ፡ አምላክ፡ አዳም፡ እንበለ፡ ይሰመይ፡ ስመ።</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.3">
+<locus from="149r"/>
+<title xml:lang="en" type="complete" ref="LIT2965RepCh246">To Gabriel</title>
+<incipit xml:lang="gez">በሰላመ፡ ገብርኤል፡ መልአክ፡ በላዕለ፡ ማርያም፡ ዘአዕረፈ። ከመ፡ እዜኑ፡ ኅዳጠ፡ ወአኮ፡ ትሩፈ።</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+</msContents>
+<physDesc>
+  <objectDesc form="Codex">
+      <supportDesc>
+          <support>
+              <material key="parchment"/>
+          </support>
+          <extent>
+              <measure unit="leaf">128</measure>
+              <measure unit="leaf" type="blank">1</measure>
+<dimensions type="outer" unit="in">
+<height>11</height>
+<width>8.375</width>
+</dimensions>
+            </extent>
+
+          </supportDesc>
+
+            <layoutDesc>
+                <layout columns="2" writtenLines="17"></layout>
+          </layoutDesc>
+          </objectDesc>
+          <handDesc>
+                  <handNote script="Ethiopic" xml:id="h1">
+                  <desc>Written in a fine hand, of the 18th Century.</desc>
+                <date notBefore="1700" notAfter="1800" resp="PRS10171Wright"/>
+            </handNote>
+          </handDesc>
+
+<decoDesc>
+<decoNote type="miniature" xml:id="d1">
+<locus target="#43v"/>
+<desc>A miniature depicting the saints <persName ref="PRS9162TaklaHa">Takla Haymānot</persName> and <persName ref="PRS4306GabraMa">Gabra Manfas Qǝddus</persName>.
+A person (possibly the donor) is placed at Takla Haymānot's feet.</desc>
+</decoNote>
+
+<decoNote type="miniature" xml:id="d2">
+<locus target="#67r"/>
+<desc>A miniature of <term key="CovenantMercy">the covenant of mercy</term>.</desc>
+</decoNote>
+
+</decoDesc>
+
+<additions>
+<list>
+
+<item xml:id="a1">
+<locus from="155r"/>
+<desc type="GuestText"><title ref="LIT4145Onausp"></title></desc>
+<q xml:lang="gez"><seg part="I">ሐሳበ፡ ጽንዕት፡ ወዘጽንጽንት፡ ፲ወ፭ለመስከረም፡ ጽንዕት፡ ወእምኔሁ፡ ጽንጽንት፡ ፳ወ፫ለጥቅምት፡ ጽንዕት፡
+  ወዘተርፈ፡ ጽንጽንት፡ ፱ለኅዳር፡ ጽንዕት፡ ወዘተርፈ፡ ጽንጽት፡</seg></q>
+</item>
+
+<item xml:id="a2">
+<locus from="1r"/>
+<desc type="OwnershipNote"/>
+<note>The name of the owner <persName ref="PRS13405HensaHaymanot">Ḥǝnṣā Hāymānot</persName> e. g. on <locus target="#1r">f. 1r</locus>.</note>
+</item>
```

Comment:
are you sure that this is an ownership note, or is the name is just mentioned in supplication or similar?

Reply:
if it might be just a mention in the text, you might simply take out the type, put e1 as an id, an put the text from the note into desc.